### PR TITLE
Adding "parent-add-adult-details" page

### PR DESCRIPTION
**Pull request by sree-cfa**
#### 🔗 Pivotal Tracker ticket
PT#186901269

#### ✍️ Description
- Adding "parent-add-adult-details" page
- Consolidate some duplicate message properties (might be something we integrate back into FFL)

Follow-up
- Delete functionality for parent list
- Mixpanel tracking
- Date helper-text (requires some a11y discussion)

#### 📷 Design reference
[Figma link](https://www.figma.com/file/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Full-Flow-(WIP)?type=design&node-id=1190-41094&mode=design&t=iWKag0lNjgyL8vGi-0)

#### ✅ Completion tasks

- [x] Added relevant tests
- [x] Meets acceptance criteria

### CODE REVIEW

File: src/main/java/org/ilgcc/app/inputs/Gcc.java [hunk 1]
```diff
@@ -119,6 +119,16 @@ public class Gcc extends FlowInputs {
 
     private String hasAdultDependents;
 
+    @NotBlank(message = "{errors.provide-first-name}")
+    private String adultDependentFirstName;
+
+    @NotBlank(message = "{errors.provide-last-name}")
+    private String adultDependentLastName;
```
Comment (guiburato) [inline src/main/java/org/ilgcc/app/inputs/Gcc.java:126]: For me, a good practice now is to always declare variables using the screen's "first" name. In this case, the "parentAddAdultDependentFirstName".

**Review by guiburato — APPROVED** [review]
lgtm

**Comment by sree-cfa** [comment]
@guiburato appreciate the feedback! This is still in draft mode tho so I'm not quit ready for it to be merged in

**Comment by guiburato** [comment]
Can you add a test?

**Comment by guiburato** [comment]
I just noticed it...

**Pull request merged**